components/phpunit_bridge.rst

```diff
@@ -220,6 +220,9 @@ following listener in your PHPUnit configuration:
 As a result, the following is guaranteed to work and is no longer a transient
 test::
 
+    # ClockMock mocks PHP functions on the namespace level
+    namespace Tests\Symfony\Component\Stopwatch;
```

Comment:
let's use something like `Tests\App` instead

Reply:
If we will use other namespace than `Tests\Symfony\Component\Stopwatch` the test will fail. 

According to https://symfony.com/doc/2.8/components/phpunit_bridge.html#troubleshooting:
> The @group time-sensitive works "by convention" and assumes that the namespace of the tested class can be obtained just by removing the Tests\ part from the test namespace.

In our example, we need to mock the `sleep` function from `Symfony\Component\Stopwatch` namespace because we're using `$stopwatch = new Stopwatch();`

See also: https://github.com/symfony/phpunit-bridge/blob/2.8/ClockMock.php#L86